app/main.py

```diff
@@ -1,3 +1,69 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> None:
```

Comment:
The return type annotation for the `__add__` method is incorrect. It should return a `Distance` object instead of `None`. Change the return type from `None` to `Distance`.